src/thumbnailTrack.ts

```diff
@@ -0,0 +1,35 @@
+import { MakeRequired } from './utils';
+
+/**
+ * Represents a custom subtitle track source that can be added to `SourceConfig.subtitleTracks`.
+ */
+export interface ThumbnailTrack {
+  /**
+   * The URL to the timed file, e.g. WebVTT file.
+   */
+  url?: string;
+  /**
+   * The label for this track.
+   */
+  label?: string;
+  /**
+   * The unique identifier for this track. If no value is provided, a random UUIDv4 will be generated for it.
+   */
+  identifier?: string;
+  /**
+   * If set to true, this track would be considered as default. Default is `false`.
+   */
+  isDefault?: boolean;
```

Comment:
```ts
{
       thumbnailTrack: {
          url: 'https://cdn.bitmovin.com/content/assets/art-of-motion-dash-hls-progressive/thumbnails/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.vtt',
          label: 'ThumbnailsTrack',
        }
}
```

```ts
{
        thumbnailTrack: 'https://cdn.bitmovin.com/content/assets/art-of-motion-dash-hls-progressive/thumbnails/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.vtt'
}
```

Reply:
Agree with @matamegger, you can ignore the `isDefault`.
Even the label can be ignored if we would use a constant value always.
Since there can be only one thumbnail track, having a label doesn't really add much value.